backend/src/main/resources/application.yaml

```diff
@@ -16,6 +16,9 @@ spring:
         default_schema: public # Using the public schema for now, so we don't have to add a manual step to the deploy process
   liquibase:
     default-schema: ${spring.jpa.properties.hibernate.default_schema}
+  jackson:
+    serialization:
+      FAIL_ON_EMPTY_BEANS: false
```

Comment:
I guess there are reasons for this not to be the default, but we do seem to need to override it a shocking percentage of the time.